rats-tls/dist/rpm/rats_tls.spec.in

```diff
@@ -61,22 +52,21 @@ if [ ! -d "$SGX_SDK" ]; then
 fi
 
 pushd %{name}
-make SGX=1
+cmake -H. -Bbuild
```

Comment:
i suggest we can input a parameter here to specify the build mode such as "sgx" "tdx" or "occlum" etc.